doc/contribute-to-core-lightning/release-checklist.md

```diff
@@ -105,3 +105,23 @@ Here's a checklist for the release process.
 3. Look through PRs which were delayed for release and merge them.
 4. Close out the Milestone for the now-shipped release.
 5. Update this file with any missing or changed instructions.
+
+## Performing the Point (hotfix) Release
+
+1. Create a new branch named `release-<VERSION>.<POINT_VERSION>`.
```

Comment:
I had wondered why the branches are not named like `release-<VERSION>.y` where commits are added to the long-lived branch over time and tags happen along that branch like `v24.08.1` and `v24.08.2` etc. The current point release process is confusing because `release-24.08.1` existed as a branch prior to release making it unclear for those following the branch if was released or if they're running an incomplete version of the branch.

Reply:
It works as is: changes are supposed to be cherry-picked from master, so generally (and this has been an exception for v24.08.2) the branch should not be long-lived.